constants/members.json

```diff
@@ -464,15 +472,15 @@
     },
     {
       "name": "William Tran",
-      "role": "developer",
-      "term": 1235,
-      "teams": ["focus-on-nature"],
+      "role": "project lead",
+      "term": 1236,
+      "teams": ["ma"],
```

Comment:
assuming you mean "mp" here haha, once we confirm the team name + id pairs you can update